W-20020689 | HL7 Connector v4.3.0 (#3068)
mulesoft/docs-connectors: hl7/4.3/modules/ROOT/pages/hl7-connector-reference.adoc

In either case the error is reported in an ERR segment. |  false |
| Fail when segment out of order in message set a| Boolean a| Whether to reject a message when a segment in a message set is out of order:

* If `true`, a message with this error is rejected.
* If `false` and the segment can be reordered, the message is accepted.

In either case the error is reported in an ERR segment. |  false |
| Fail when unused segment included in message set a| Boolean a| Whether to reject a message when a segment marked as unused is included in a message set: 

* If `true`, a message with this error is rejected.
* If `false`, the message is accepted and the unused segment is ignored.

In either case the error is reported in an ERR segment. |  false |
| Fail when too many repeats of segment a| Boolean a| Whether to reject a message when a segment occurs too many times in a message set:

* If `true`, a message with this error is rejected.
* If `false`, the message is accepted.

In either case, the error is reported in an ERR segment. |  false |
| HL7 Message segment terminator a| Enumeration, one of:

** CR
** LF |  Value for the segment terminator. |  CR |

> | Handle empty arrays for fields in message a| Boolean a| When an incoming payload has empty arrays and the parameter is `true`, the connector emits field separation character for optional repeatable fields.

Reviewer: ```suggestion
| Handle empty arrays for fields in message a| Boolean a| Emits the field separation character for optional repeatable fields when the incoming payload contains empty arrays and the `Handle empty arrays for fields in message` parameter is set to *true*.
```
